File: 605. Can Place Flowers/solution.java
class Solution {
  public boolean canPlaceFlowers(int[] flowerbed, int n) {
    int slots = 0;
    int count = 1;
    for (int index = 0; index < flowerbed.length; index++){
      int flower = flowerbed[index];
      if (flower == 0){
        count += 1;
      }
      else{
        slots += (count - 1) / 2;
        count = 0;
      }
    }

    return (slots + (count / 2)) >= n;
  }
}

File: 605. Can Place Flowers/solution.py
class Solution(object):
  def canPlaceFlowers(self, flowerbed, n):
    """
    :type flowerbed: List[int]
    :type n: int
    :rtype: bool
    """
    slots = 0
    count = 1
    for flower in flowerbed:
      if flower == 0: 
        count += 1
      else:
        slots += (count - 1) / 2
        count = 0
    return (slots + (count / 2)) >= n